assessment.md

```diff
@@ -143,39 +143,96 @@ TODO: add description for each assessment level for this skill.
 
 * Requirements engineering
 
-TODO: add description for each assessment level for this skill.
+Requirements engineering will be evaluated on this scale:
+
+* None = Fails to adds any requirements for the program and shows no knowledge of what the program is doing
+* Inadequate = Adds one requirement for the program
+* Adequate = Adds two requirements for the program
+* Good = Adds three requirements for the program
+* Excellent = Adds more than three requirements for the program and shows knowledge of what the program is doing
 
 * Software architecture
 
-TODO: add description for each assessment level for this skill.
+The architecture of the software will be evaluated on this scale:
+
```

Comment:
These two lines can be removed for consistency with other sections of the document and other branches